sources/reddit_test.go

```diff
@@ -0,0 +1,75 @@
+package sources_test
+
+import "fmt"
+import "io/ioutil"
+import "github.com/bobo333/sponge/shared"
+import "github.com/bobo333/sponge/sources"
+import "github.com/stretchr/testify/assert"
+import "net/http"
+import "net/http/httptest"
+import "os"
+import "testing"
+
+func TestGetReddit(t *testing.T) {
+	bytes, _ := ioutil.ReadFile("reddit.txt")
```

Comment:
this might be guaranteed not to fail... or it really might not if anything goes awry with open file handles or someone moves the test file away or permissions are whack.  What you'll see is a test failure somewhere else when really it was a failure to read this file very early in the test and you'll be confused.  This is unlikely to be a problem in a solo project but imagine yourself in this situation after the torch has been handed to three other people before you got it and all of a sudden someone's doing something with a new jenkins AMI and the file can't be read even though it's there.

Moral of the story: if it can return an error, check it -- especially in your tests.